postman_tests/run_tests.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import
```

Comment:
For this whole file, this file should be able to signal failures to parent process if any of the test steps fail/raise errors